gpytorch/functions/__init__.py

```diff
@@ -12,5 +12,7 @@
 ]
 
 fastest = True
+fast_pred_var = True
```

Comment:
In my opinion, let's make this False by default for a little while until the Lanczos stuff has been tested on kernels other than Matern and RBF.

Reply:
Fixed this.